opencti-platform/opencti-front/src/private/components/cases/case_incidents/CaseIncidentDetails.tsx

```diff
@@ -233,7 +233,7 @@ const CaseIncidentDetails: FunctionComponent<CaseIncidentDetailsProps> = ({
           {t('Correlated cases')}
         </Typography>
         <List classes={{ root: classes.relatedContainers }}>
-          {expandable
+          {(data.relatedContainers?.edges && data.relatedContainers.edges.length >= 2)
```

Comment:
Not sure, let's talk about that

Reply:
You need to extract the filtering of the list before the return and check that this computed list as at least one element